### DISCUSSION TITLE
Feature request: Multi value support in Combobox/Select

### DISCUSSION THREAD

**Discussion by jdpnielsen** [umbraco/Umbraco.UI#396] [Ideas]
Umbraco's current dropbox implementation supports multi-value selection. It would be great if uui could support a similar feature.

**Comment by AndersBrohus** [comment]
Yeah this would be great to have support for "multiple", and perhabs an eaiser implementation of the uui-combobox.. :)

**Comment by bjarnef** [comment]
It could eventually be a separate component similar to this:
https://sap.github.io/ui5-webcomponents/components/MultiComboBox/

But I think it would be nicer if same component can be use and just based on `multiple` attribute or property as in Fluent UI via a `multiselect` property:
https://react.fluentui.dev/?path=/docs/components-combobox--docs
